code/modules/mob/living/simple_animal/hostile/killertomato.dm

```diff
@@ -10,7 +10,7 @@
 	maxHealth = 30
 	health = 30
 	see_in_dark = 3
-	butcher_results = list(/obj/item/reagent_containers/food/snacks/meat/slab/killertomato = 2)
```

Comment:
Is this an intentionally unstated change or a leftover of the early phases of this branch?

Reply:
Actually... an oversight.

Reply:
Oh. It's actually a fuckup on my part. The hunter tomatoes were meant to be the ones which don't drop stuff.